keps/sig-node/1769-memory-manager/README.md

```diff
@@ -577,96 +595,99 @@ the health of the service?**
   
   *For a pod author:*
   
-  * Pod succeeded to start. You have two options to verify if containers pinned to the NUMA node
+  * Pod succeeded to start. You have two options to verify if containers are pinned to the NUMA node
```

Comment:
realizing now that pod authors have no way to actually see these things.  Is there any representation that a namespace editor can see via the API?